packages/transactions/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -187,3 +189,11 @@ export const OutboundShipmentDetailView: FC = () => {
     </TabContext>
   ) : null;
 };
+
+export const OutboundShipmentDetailView: FC = () => {
+  return (
+    <TableProvider createStore={createTableStore}>
+      <OutboundShipmentDetailViewComponent />
+    </TableProvider>
+  );
+};
```

Comment:
Now I've added the `useTableStore` into the `RemoteDataTable` the context becomes mandatory` 🤔

Reply:
have you tried with two tables on a page? that would work, right? having a different provider wrapped around each.

I'm concerned that any component which uses a data table has to remember to wrap it in a provider, though I think I'm worrying about nothing and it is probably fine